util/Migrator/DbMigrator.cs

```diff
@@ -24,6 +25,8 @@ public DbMigrator(string connectionString, ILogger<DbMigrator> logger)
     }
 
     public bool MigrateMsSqlDatabaseWithRetries(bool enableLogging = true,
+        bool repeatable = false,
+        string folderName = "DbScripts",
```

Comment:
🎨 Please create a new `Constants.cs` next to this file. It can be a static class named `Constants` with some ... constant strings in it, like this one e.g. `FolderName`. You can reuse it below.